lib/calc/xmedian.c

```diff
@@ -44,10 +44,10 @@ static int dcmp(const void *aa, const void *bb)
 
 int f_median(int argc, const int *argt, void **args)
 {
-    static void *array;
-    static int alloc;
     int size = argc * Rast_cell_size(argt[0]);
     int i, j;
+    const int threshold = 32;
```

Comment:
It seems you need to make it a 'define' rather than const int to be recognized as compile-time constant and the array be truly plain array rather than variable length array (VLA).